wdl/ResolveCpxSv.wdl

```diff
@@ -329,30 +336,36 @@ task ResolvePrep {
       > noref.vcf.gz
     rm -f header.vcf
 
-    #Third, use remote tabix to pull down the discfile chunks within ±2kb of all
+    #Third, use GATK to pull down the discfile chunks within ±2kb of all
     # INVERSION breakpoints, and bgzip / tabix
-    echo "Forming regions.tabix.bed"
+    echo "Forming regions.bed"
     { grep -Ev "^#" input.bed || true; } \
       | (fgrep INV || printf "") \
       | awk -v OFS="\t" -v buffer=2000 \
         '{ print $1, $2-buffer, $2+buffer"\n"$1, $3-buffer, $3+buffer }' \
       | awk -v OFS="\t" '{ if ($2<1) $2=1; print $1, $2, $3 }' \
       | sort -Vk1,1 -k2,2n -k3,3n \
       | bedtools merge -i - \
-      > regions_to_tabix.bed
+      > regions.bed
 
-    if [ -s regions_to_tabix.bed ]; then
+    if [ -s regions.bed ]; then
       echo "Localizing all discfile shards..."
-      export GCS_OAUTH_TOKEN=`gcloud auth application-default print-access-token`
       DISC_FILE_NUM=0
       while read GS_PATH_TO_DISC_FILE; do
         ((++DISC_FILE_NUM))
         SLICE="disc"$DISC_FILE_NUM"shard"
 
-        tabix -R regions_to_tabix.bed "$GS_PATH_TO_DISC_FILE" \
+        java -Xmx~{java_mem_mb}M -jar ${GATK_JAR} LocalizeSVEvidence \
+              --sequence-dictionary ~{ref_dict} \
+              --evidence-file $GS_PATH_TO_DISC_FILE \
+              -L regions.bed \
+              -O ${SLICE}.txt
```

Comment:
If you wanted to, could you change this to `-O /dev/stdout` and keep the pipe as it was?

Reply:
Hm that makes me nervous in case a log message or something gets sent to stdout. I'd rather keep it this way to be safe.